blazor/diagram/how-to.md

```diff
@@ -1,17 +1,17 @@
 ---
 layout: post
-title: Diagram Methods in Blazor Diagram Component | Syncfusion
+title: Diagram Methods and Properties in Blazor Diagram Component | Syncfusion
 description: Checkout and learn here all about diagram methods support in Syncfusion Blazor Diagram component, it's elements and more.
 platform: Blazor
 control: DiagramComponent
 documentation: ug
 ---
 
-# Diagram Methods in Blazor Diagram Component
+# Diagram Methods and Properties in Blazor Diagram Component
```

Comment:
How to achieve common use cases using diagram methods and properties in blazor diagram coponent.